Estranged.Lfs.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
-VisualStudioVersion = 16.0.29806.167

```

Comment:
The changes to this file don't look necessary

Reply:
Ok, reverting this change